.buildkite/publish-ruby-gems.sh

```diff
@@ -51,15 +51,25 @@ with_backoff() {
 # sorbet-static, but the sorbet-static gem push failed.
 #
 # (By failure here, we mean that RubyGems.org 502'd for some reason.)
-if ! gem list --remote rubygems.org --exact 'sorbet-static' | grep -q "$release_version"; then
-  for gem_archive in "_out_/gems/sorbet-static-$release_version"-*.gem; do
-      with_backoff gem push --verbose "$gem_archive"
-  done
+# Push the linux gem
+if ! gem fetch sorbet-static --platform x86_64-linux --version "$release_version" | grep -q "ERROR"; then
+  with_backoff gem push --verbose "_out_/gems/sorbet-static-$release_version-x86_64-linux.gem"
+fi
+
+# Push the mac gem
+if ! gem fetch sorbet-static --platform universal-darwin --version "$release_version" | grep -q "ERROR"; then
+  with_backoff gem push --verbose "_out_/gems/sorbet-static-$release_version-universal-darwin-"*.gem
```

Comment:
seems like this broke publishing
https://buildkite.com/sorbet/sorbet/builds/7970#54d3f6d0-8c3d-40f3-9bca-67fb77d24344

Reply:
Oh cause Darwin has multiple. 
I can fix this asap.